### PR TITLE
The first time making a pull request,hoo....

### PR DESCRIPTION
**Pull request by mingzhaochina**
Dear Perol:

Thank you for sharing the ConvNetQuake code!We trained our own model based on more than 20000 events and over 60000 noise of hand-picked wenchuan aftershocks in Sichuan network,China.Our model achieves 99% train accuracy and 94% validation accuracy.And we classify one day's continuous waveform of one station in Sichuan(MXI) as an test to the model. The results seem to be quite satisfactory:in the day we have 43 obvious earthquakes and about 240 microearthquakes,the model can detect  all 43 obvious earthquakes and most of microearthquakes,with relatively fewer wrong detects than traditional method (about 230 false positives ).

Besides,the code is based on an old version of tensorflow,we made some tiny change so that it can work in tensorflow 1.4 or higher.

Based on that I try to make a pull request to you.I hope you consider about it.

Thank you again for sharing this interesting code,I have had a lot of fun when trying it.

Best regards
Ming

### CODE REVIEW

**Comment by txpdev** [comment]
Hi @mingzhaochina, thanks for using convnetquake and your contribution. I see that you are pushing some txt and png files as well as checkpoints. I would rather keep the repo clean and not tailored to a specific project/location so that it can be used easily to other projects. Please only submit requests that concern the code.

**Comment by mingzhaochina** [comment]
Yes,I understand that. I will not do that again^_^

Thanks!

Ming Zhao

2018-04-15 22:44 GMT+08:00 Thibaut Perol <notifications@github.com>:
